WordPress/src/main/java/org/wordpress/android/util/DialogUtils.java

```diff
@@ -11,22 +14,55 @@
 import org.wordpress.android.widgets.OpenSansEditText;
 import org.wordpress.android.widgets.WPTextView;
 
-public class DialogUtils {
-    public static void showMyProfileDialog(Context context,
-                                           String title,
-                                           String initialText,
-                                           String hint,
-                                           boolean isMultiline,
-                                           final Callback callback) {
-        LayoutInflater layoutInflater = LayoutInflater.from(context);
+public class DialogUtils extends DialogFragment {
```

Comment:
`DialogUtils` extends a Fragment now so, better rename it to something more relevant. Cheers!